expat/CMakeLists.txt

```diff
@@ -412,23 +413,38 @@ if(EXPAT_BUILD_FUZZERS)
             "-DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++.")
     endif()
 
-    string(FIND "${CMAKE_EXE_LINKER_FLAGS}" "-fsanitize" sanitizer_present)
-    if(${sanitizer_present} EQUAL "-1")
+    if(EXPAT_OSSFUZZ_BUILD AND NOT EXPAT_BUILD_FUZZERS)
```

Comment:
This line and all further down are within a block of `if(EXPAT_BUILD_FUZZERS)` so all further checks for `EXPAT_BUILD_FUZZERS` can (a) be dropped or (b) be made about `EXPAT_OSSFUZZ_BUILD`, instead.

Reply:
Ah right, this is now redundant. Thank you.